plrust/build

```diff
@@ -52,9 +52,18 @@ fi
 cargo update -p pgrx
 cargo fetch
 if [ "$CI" != true ]; then
-    cargo install cargo-pgrx \
-     --version "0.10.0" \
-     --locked # use the Cargo.lock in the pgrx repo
+    # Attempt to get pgrx version from cargo tree. Looking for a pattern such as "pgrx v0.10.0"
+    PGRX_VERSION=$(cargo tree | grep -E "\s*pgrx\s+v[0-9]+\.[0-9]+\.[0-9]+" | head -n 1 | cut -f2- -dv)
```

Comment:
Just realized that this should maybe be `cargo tree --package plrust`. Thoughts?

Reply:
yeah, probably.  I think the thing I send you on Discord assumed being in the `plrust/` crate directory.  Testing manually it doesn't seem to matter tho.

Could also add `--depth 1`